modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss

```diff
@@ -0,0 +1,30 @@
+@import 'atlas-variables';
+
+$controlMenuHeight: 56px;
+
+.knowledge-base-portlet-admin {
+	.kbVerticalBar {
+		> div {
+			height: 100%;
+			min-height: calc(100vh - #{$controlMenuHeight});
```

Comment:
Why we need `height` and `min-height`?

We always want to fit the whole height.

For me, only height works fine

```scss
height: calc(100vh - #{$controlMenuHeight});
```

But maybe I missing something.

Reply:
Sometimes the viewport is shorter that the page content (like in mobile). Similar thing is done with the [info panel](https://github.com/ambrinchaudhary/liferay-portal/blame/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js#L1057-L1057). You can see the difference in the next screenshot (I know the panel overflows the control menu, not sure if it's a component bug, will dig into it)

Only with height: 

<img width="688" alt="height" src="https://user-images.githubusercontent.com/8373764/179755521-6caeffcb-7399-472f-bae8-4d5511ff39d9.png">

With min-height: 
<img width="706" alt="min-height" src="https://user-images.githubusercontent.com/8373764/179755591-7fd638c6-9a09-4541-9791-f11069922dac.png">

Reply:
Thanks for the explanation ❤️